src/main/java/org/myrobotlab/service/InMoov.java

```diff
@@ -1519,29 +1637,30 @@ public void preShutdown() {
     if (RightRelay1 != null) {
       RightRelay1.off();
     }
-    // TODO better thing to detect connected arduinos
-    // we cant use arduino.stopService()
-    // ServoController don't have serials :P - new way must be figured out
-    if (rightHand != null) {
-      // rightHand.arduino.serial.disconnect();
-      // rightHand.arduino.serial.stopRecording();
-      // rightHand.arduino.disconnect();
-    }
-    if (leftHand != null || head != null) {
-      // leftHand.arduino.serial.disconnect();
-      // leftHand.arduino.serial.stopRecording();
-      // leftHand.arduino.disconnect();
-    }
   }
 
   @Override
   public void startService() {
     super.startService();
+    //we need auto load : if user launch inmoov from runtime, or start from script
+    //without load : this will overwrite EVERY previous personal configs !!
+    //OR : we need to disable autosave EVERYWHERE..
+    load();
```

Comment:
Kwatters… That's a fact if we launch inmoov service from runtime GUI load **will not be called**, and every parameters will be overwriten at shutdown… because we don't want gui for now, I will revert to avoid infinite loop talk, maybe we can talk about it later